### PR TITLE
n: update to 9.2.0

### PR DESCRIPTION
**Pull request by akierig**
#### Description

n: update to 9.2.0

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 13.6 22G120 arm64
Xcode 15.0 15A240d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/n/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tj n 9.1.0 v
+github.setup        tj n 9.2.0 v
 categories          devel
 platforms           darwin
```
Comment (herbygillot) [inline devel/n/Portfile:8]: `platforms darwin` is implicit in MacPorts now and can be dropped

File: devel/n/Portfile [hunk 2]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tj n 9.1.0 v
+github.setup        tj n 9.2.0 v
```
Comment (herbygillot) [inline devel/n/Portfile:6]: Please add a `revision` field as well:

```suggestion
github.setup        tj n 9.2.0 v
revision            0
```

**Comment by akierig** [comment]
thanks @herbygillot got both of those.

**Comment by herbygillot** [comment]
Thanks @akierig

**Comment by akierig** [comment]
the pleasure and privilege is mine, thank you @herbygillot

**Pull request merged**